packages/web-scripts/src/integration.test.ts

```diff
@@ -60,7 +60,8 @@ const SETUP_REPO_TIMEOUT = 30000;
 const TEST_SCRIPTS_TIMEOUT = 60000;
 // const GITHUB_URL = 'https://github.com/spotify/web-scripts.git';
 
-describe('integration tests', () => {
+// eslint-disable-next-line jest/no-disabled-tests
+describe.skip('integration tests', () => {
```

Comment:
not needed anymore?

Reply:
it's flakey. sometimes it works locally but it works in CI. sometimes it works in CI but not locally. Sometimes it works if you pass an extra flag. I hate it and it's unreliable. But I'd like to maybe revisit rewriting another, less complex (if possible) integration test and leave the old one there as a reference